contrib/libvirt/template.py

```diff
@@ -0,0 +1,136 @@
+pkgname = "libvirt"
+pkgver = "10.2.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dattr=enabled",
+    "-Dblkid=enabled",
+    "-Dcapng=enabled",
+    "-Dcurl=enabled",
+    "-Ddriver_qemu=enabled",
+    "-Ddriver_secrets=enabled",
+    "-Dfirewalld=enabled",
+    "-Dfirewalld_zone=enabled",
+    "-Dfuse=enabled",
+    "-Dlibiscsi=enabled",
+    "-Dlibnl=enabled",
+    "-Dlibssh2=enabled",
+    "-Dno_git=true",
+    "-Dnumactl=enabled",
+    "-Dpackager=Chimera Linux",
+    "-Dpciaccess=enabled",
+    "-Dstorage_dir=enabled",
+    "-Dstorage_disk=enabled",
+    "-Dstorage_fs=enabled",
+    "-Dstorage_lvm=enabled",
+    "-Dstorage_mpath=enabled",
+    "-Dstorage_scsi=enabled",
+    "-Dstorage_zfs=enabled",
+    "-Dudev=enabled",
+    "-Dyajl=enabled",
+    f"-Dpackager_version={pkgver}",
+    "-Dqemu_user=virtqemu",
+    "-Dqemu_group=virtqemu",
+]
+hostmakedepends = [
+    "libxml2-progs",
+    "lvm2",  # buildtime check
+    "meson",
+    "mkfs",  # buildtime check
+    "mount",  # buildtime check
+    "perl",
+    "pkgconf",
+    "python-docutils",
+    "xsltproc",
+]
+makedepends = [
+    "acl-devel",
+    "attr-devel",
+    "bash-completion",
+    "device-mapper-devel",
+    "fuse-devel",
+    "glib-devel",
+    "gnutls-devel",
+    "libcap-ng-devel",
+    "libcurl-devel",
+    "libiscsi-devel",
+    "libnl-devel",
+    "libnuma-devel",
+    "libpcap-devel",
+    "libpciaccess-devel",
+    "libsasl-devel",
+    "libssh-devel",
+    "libssh2-devel",
+    "libtirpc-devel",
+    "libxml2-devel",
+    "linux-headers",
+    "parted-devel",
+    "polkit-devel",
+    "readline-devel",
+    "udev-devel",
+    "yajl-devel",
+]
+checkdepends = [
+    "pahole",
+    "python-black",
+    "python-flake8",
+    "python-pytest",
+]
+depends = [
+    "dnsmasq",
+]
+pkgdesc = "API, daemon, and management tool for virtualization"
+maintainer = "cesorious <cesorious@gmail.com>"
+license = "LGPL-2.1-only"
+url = "https://libvirt.org"
+source = f"https://download.libvirt.org/libvirt-{pkgver}.tar.xz"
+sha256 = "215772bc5dc4a672e67ffa9de3774f05ed4b7ed282dbe296ec5c9fec01dd7ae3"
+options = ["!lto"]
+
+
+def post_install(self):
+    self.rm(self.destdir / "usr/lib/sysusers.d/libvirt-qemu.conf")
+    self.install_file(
+        self.files_path / "sysusers.conf",
+        "usr/lib/sysusers.d",
+        name="libvirt.conf",
+    )
+    self.install_service(self.files_path / "libvirtd")
+    for service in [
+        "ch",
+        "interface",
+        "lock",
+        "log",
+        "lxc",
+        "network",
+        "nodedev",
+        "nwfilter",
+        "proxy",
+        "qemu",
+        "secret",
+        "storage",
+        "vbox",
+    ]:
+        self.install_service(self.files_path / f"virt{service}d")
+
+
+@subpackage("libvirt-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("libvirt-firewalld")
```

Comment:
for whatever reason the libvirt-firewalld subpackage only grabs `usr/lib/firewalld/policies/libvirt-routed-in.xml` and none of the other zones or policies.

Reply:
the first return is all that happens, you can't return from a function multiple times here since it's not a generator (afaik)

just write out the whole list of five items

Reply:
I tried that too, but it still only returns the first file

Reply:
doing  

```
return [
	"a",
	"b",
	"c",
]
```

definitely returns all the files

Reply:
```
return [
    "usr/lib/firewalld/policies/libvirt-routed-in.xml",
    "usr/lib/firewalld/policies/libvirt-routed-out.xml",
    "usr/lib/firewalld/policies/libvirt-to-host.xml",
    "usr/lib/firewalld/zones/libvirt-routed.xml",
    "usr/lib/firewalld/zones/libvirt.xml",
]
```

there you go

Reply:
or probably just `return ["usr/lib/firewalld"]` unless you're keeping other files in there in the main package

Reply:
woops, forgot  to reinstall the libvirt package after rebuilding...